client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
+	width: 100%;
+	margin: 8px 0;
+}
+
+.reset-password-confirm-form__description {
+	margin: 4px 0;
+}
+
+.form-label.reset-password-confirm-form__text-input-label {
+	display: inline;
+	float: left;
```

Comment:
Introducing `FormPasswordInput` eliminated the need of this together. Thanks for introducing a good chance to learn flex for me, though :)